app/test_main.py

```diff
@@ -1,3 +1,21 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,expected",
+    [
+        pytest.param(0, 0, [0, 0]),
+        pytest.param(14, 14, [0, 0]),
```

Comment:
The test case for `cat_age=14` and `dog_age=14` is essentially the same as `cat_age=0` and `dog_age=0`, both resulting in `[0, 0]`. According to the checklist, make sure that your tests don't test the same case multiple times. Consider removing or replacing this test case with a more unique scenario.